src/Ansatz/Chain.jl

```diff
@@ -343,6 +343,51 @@ function isrightcanonical(qtn::Chain, site; atol::Real = 1e-12)
     return isapprox(contracted, identity_matrix; atol)
 end
 
+canonize(tn::Chain, args...; kwargs...) = canonize!(deepcopy(tn), args...; kwargs...)
+canonize!(tn::Chain, args...; kwargs...) = canonize!(boundary(tn), tn, args...; kwargs...)
+
+"""
+canonize(boundary::Boundary, tn::Chain)
+
+Transform a `Chain` tensor network into the canonical form (Vidal form), that is,
+we have the singular values matrix Λᵢ between each tensor Γᵢ₋₁ and Γᵢ.
+"""
+function canonize!(::Open, tn::Chain)
+    S_vals = Tensor[]
```

Comment:
Maybe we can just rename it to `Λ`?
```suggestion
    Λ = Tensor[]
```